packages/application-config/test/process-config.spec.js

```diff
@@ -26,7 +26,10 @@ beforeEach(() => {
 
 describe('processing a simple config', () => {
   beforeEach(() => {
-    loadConfig.mockReturnValue(fixtureConfigSimple);
+    loadConfig.mockReturnValue({
```

Comment:
This function now returns not only the configuration but also the filename, so we need to update all these mocks.